src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -0,0 +1,344 @@
+import classnames from 'classnames';
+import { useCallback, useMemo, useState } from 'preact/hooks';
+
+import { ArrowLeftIcon, ArrowRightIcon } from '../../../../components/icons';
+import { IconButton, InputGroup } from '../../../../components/input';
+import Select from '../../../../components/input/SelectNext';
+import Library from '../../Library';
+
+const items = [
+  { id: '1', name: 'All students' },
+  { id: '2', name: 'Albert Banana' },
+  { id: '3', name: 'Bernard California' },
+  { id: '4', name: 'Cecelia Davenport' },
+  { id: '5', name: 'Doris Evanescence' },
+];
+
+function Select_({
```

Comment:
Maybe call this `SelectExample` or something, to make the difference with the imported `Select` more obvious.

Reply:
Yep. I initially used `Select_` so that we could show code in the examples, but I ended up manually writing the example because the part about providing props was more important.